tools/interu/src/config/mod.rs

```diff
@@ -0,0 +1,186 @@
+use std::{collections::BTreeMap, fmt::Display, path::Path};
+
+use rand::{distributions::WeightedIndex, prelude::Distribution as _, thread_rng};
+use serde::{Deserialize, Serialize};
+use snafu::{OptionExt, ResultExt, Snafu};
+use tracing::instrument;
+
+use crate::{
+    config::{
+        profile::{Profile, StrategyValidationError, TestRun},
+        runner::{ConvertNodeGroupError, Distribution, ReplicatedNodeGroup, Runner},
+    },
+    instances::Instances,
+};
+
+pub mod profile;
+pub mod runner;
+
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display("failed to read file"))]
+    ReadFile { source: std::io::Error },
+
+    #[snafu(display("failed to deserialize file"))]
+    Deserialize { source: serde_yaml::Error },
```

Comment:
This error variant is called from `serde_yaml::from_str`, so the filemane should be added to the error.

Reply:
Makes sense, I'll add it.